server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -246,7 +249,7 @@ export function verifyStorageToken(
 	};
 }
 
-export function validateTokenRevocationClaims(): RequestHandler {
+export function validateTokenClaimsScopes(expectedScopes: string): RequestHandler {
```

Comment:
>validateTokenClaimsScopes [](http://example.com/codeflow?start=16&length=25)

(nit) `validateTokenScopesClaim`?

Reply:
maybe let's keep the string parameter type for now, because I believe when new scope is moved to protocol, this will change anyway.